octodns_rackspace/__init__.py

```diff
@@ -47,17 +47,25 @@ class RackspaceProvider(BaseProvider):
     SUPPORTS = set(
         ('A', 'AAAA', 'ALIAS', 'CNAME', 'MX', 'NS', 'PTR', 'SPF', 'TXT')
     )
-    TIMEOUT = 5
 
     def __init__(
-        self, id, username, api_key, ratelimit_delay=0.0, *args, **kwargs
+        self,
+        id,
+        username,
+        api_key,
+        ratelimit_delay=0.0,
+        timeout=5,
+        *args,
+        **kwargs,
     ):
         self.log = logging.getLogger(f'RackspaceProvider[{id}]')
         super().__init__(id, *args, **kwargs)
 
         auth_token, dns_endpoint = self._get_auth_token(username, api_key)
         self.dns_endpoint = dns_endpoint
 
+        self.default_timeout = timeout
```

Comment:
It's a nit, but i'd prefer the property match the param on the off chance someone is using the class outside of the normal way it'd allow them to change the timeout property if they had reason to.

```suggestion
        self.timeout = timeout
```

Reply:
@ross No worries, I wasn't sure about the style. Completely fine to me :)